source/mapping-rules/fields/data-type-conversion-guide.txt

```diff
@@ -0,0 +1,229 @@
+.. _rm-data-type-conversion-guide:
+
+==========================
+Data Type Conversion Guide
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This page outlines examples that show the result of converting the 
+source value and data-type to a specific BSON data-type using customized 
+fields.
+
+Behavior
+~~~~~~~~
+
+Errors during a sync job result in the field being ``BsonNull``. 
+Your fields :guilabel:`Null handlings` strategy does not affect this 
+behavior.
+
+Your sync job continues with errors unless it reaches the user-defined 
+threshold.
+
+.. include:: /includes/fact-calculated-fields-type.rst
+
+Integer
+~~~~~~~
+
+The following example shows results from applying data type 
+customization for the value ``100`` with a ``Integer`` data type.
+
+.. list-table::
+   :widths: 40 60
+   :header-rows: 1
+
+   * - Conversion Type
+     - Conversion Value
+
+   * - ``Boolean``
+     - ``false``
+
+   * - ``Date``
+     - ``Thu Jan 01 10:00:00 AEST 1970``
+
+   * - ``Decimal``
+     - ``100.0``
+
+   * - ``Double``
+     - ``100.0``
+
+   * - ``Integer``
+     - ``100``
+
+   * - ``Long``
+     - ``100``
+
+   * - ``String``
+     - ``100``
+
+Boolean
+~~~~~~~
+
+The following example shows results from applying data type 
+customization for the value ``true`` with a ``Boolean`` data type.
+
+.. list-table::
+   :widths: 40 60
+   :header-rows: 1
+
+   * - Conversion Type
+     - Conversion Value
+
+   * - ``Boolean``
+     - ``true``
+
+   * - ``Date``
+     - ``Thu Jan 01 10:00:00 AEST 1970``
+
+   * - ``Decimal``
+     - ``1.0``
+
+   * - ``Double``
+     - ``1.0``
+
+   * - ``Integer``
+     - ``1``
+
+   * - ``Long``
+     - ``1``
+
+   * - ``String``
+     - ``"true"``
+
+Long
+~~~~
+
+The following example shows results from applying data type 
+customization for the value ``1233140483647`` with a ``Long`` data type.
+
+.. list-table::
+   :widths: 40 60
+   :header-rows: 1
+
+   * - Conversion Type
+     - Conversion Value
+
+   * - ``Boolean``
+     - ``false``
+
+   * - ``Date``
+     - ``Wed Jan 28 22:01:23 AEDT 2009``
+
+   * - ``Decimal``
+     - ``1233140483647.0``
+
+   * - ``Double``
+     - ``1233140483647.0``
+
+   * - ``Integer``
+     - ``Error (Too big)``
+
+   * - ``Long``
+     - ``1233140483647``
+
+   * - ``String``
+     - ``"1233140483647"``
+
+Decimal
+~~~~~~~
+
+The following example shows results from applying data type 
+customization for the value ``26.55`` with a ``Decimal`` data type.
+
+.. list-table::
+   :widths: 40 60
+   :header-rows: 1
+
+   * - Conversion Type
+     - Conversion Value
+
+   * - ``Boolean``
+     - ``false``
+
+   * - ``Date``
+     - ``Error``
+
+   * - ``Decimal``
+     - ``26.55``
+
+   * - ``Double``
+     - ``26.55``
+
+   * - ``Integer``
+     - ``26``
+
+   * - ``Long``
+     - ``26``
+
+   * - ``String``
+     - ``"26.55"``
+
+String
+~~~~~~
```

Comment:
(question)
Do we need to explain the difference tween the two string examples? What difference are you trying to highlight?